Src/Fare/RegExp.cs

```diff
@@ -71,8 +73,8 @@ private RegExp()
         ///   Initializes a new instance of the <see cref = "RegExp" /> class from a string.
         /// </summary>
         /// <param name = "s">A string with the regular expression.</param>
-        public RegExp(string s)
-            : this(s, RegExpSyntaxOptions.All)
+        public RegExp(string s, string anyCharCharset = null)
```

Comment:
docs, what is meaning of "anycharcharset"

Reply:
ok